.github/workflows/build.yml

```diff
@@ -0,0 +1,31 @@
+name: build
+
+on:
+  push:
+    branches:
+      - main
+      - master
+  pull_request:
+  workflow_dispatch:
+
+concurrency:
+  group: "${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}"
+  cancel-in-progress: true
+
+jobs:
+  build_testing:
+    runs-on: ubuntu-24.04
+    strategy:
+      fail-fast: false
+      matrix:
+        ros_distro:
+          - humble
+          - jazzy
+          - rolling
+    steps:
+      - name: Build for ROS ${{ matrix.ros_distro }}
+        uses: jspricke/ros-deb-builder-action@main
+        with:
+          DEB_DISTRO: ${{ (matrix.ros_distro == 'humble' && 'jammy') || 'noble' }}
```

Comment:
This logic is wrong and causes `noble` to be selected for `humble`, which is not a valid combination. This leads to `noble` packages being considered (`Hit:2 http://azure.archive.ubuntu.com/ubuntu noble InRelease`) for the `humble` builds: https://github.com/AndrejOrsula/pymoveit2/actions/runs/15056192600/job/42322536727.

Since this logic has to adapt for new Ubuntu and ROS releases, maybe just maintain the Ubuntu-ROS distribution combination on the matrix like so:
```yaml
matrix:
  include:
    - {ros: humble, deb: jammy}
    - {ros: jazzy, deb: noble}
```

Reply:
Sorry, I mixed up which repos are used on the runner (`ubuntu-24.04`) and which are used inside the debootstrap'ed environment. But still, I think adding the matrix like this is a bit easier to maintain for newer distros instead of adjusting the logic.